### ISSUE TITLE
1.2 has been released prematurely and 1.1 docs are missing

### ISSUE THREAD

**Issue by mattheworiordan** [ably/docs#854]
See https://github.com/ably/docs/pull/843#issuecomment-607522231

This has been merged into the website now and released too, making this more of an issue which needs to be fixed with high priority.

![screenshot_2020-04-02_08-21-58_pm](https://user-images.githubusercontent.com/43789/78293442-7759d280-7520-11ea-93c1-9cf096fa7246.png)

The 1.2 work should remain on a feature branch until launched.

**Comment by QuintinWillison** [comment]
I believe I've fixed this with reverts https://github.com/ably/docs/commit/280a99b96041025c1b2d66ee8f185708ff73bd36 and https://github.com/ably/docs/commit/e308cafce27d2651b7b08fe68e94f2e9de36cb65, in that 1.2 is now [back in an integration branch](https://github.com/ably/docs/pull/856).

I don't believe there's any more work I need to do on this issue, and I've responded to @mattheworiordan [elsewhere](https://github.com/ably/docs/pull/843#issuecomment-609580268) regarding versions folders.

**Comment by mattheworiordan** [comment]
> @QuintinWillison Can you please be specific regarding your statement "we've effectively skipped a version now in the docs".

I had stated why in https://github.com/ably/docs/pull/843#issuecomment-607522231

"This is important because if someone is now using 1.1, what docs do they refer to? The process we followed before was to simply copy the files into a 1.1 versions folder, and voila, you magically have 1.1."

Only creating a 1.1 version because it's changed is wrong for a few reasons:

- Anyone on a 1.1 SDK now has no way of viewing 1.1 docs.  Would they view 1.2 or 1.0?  How would (or should) they know.  We should just show them 1.1 docs.
- Any changes you make subsequently to 1.2 docs, which may not apply to 1.1 in future (yes, not you, but someone else may do this) will inadvertently now create a change that is potentially incompatible with 1.1
- We have no single view of what the SDK docs look like at 1.1
- Users will have a pretty broken experience if they view push docs with version 1.1, then jump to the realtime docs, and suddenly see 1.2.  Every time they jump between docs they may get a version that doesn't match the SDK they are using.

So reopening this issue as this is not resolved.
